docs/nonscalars/mrtrix_anat.csv

```diff
@@ -1,2 +1,2 @@
-\*space-T1w\*atlas-hsvs\*_dseg.nii.gz,Hybrid Surface/Voume Segmentation in MRtrix3 5tt format. Aligned in coordinate space to ``space-T1w``. 
-\*space-fsnative\*atlas-hsvs\*_dseg.nii.gz,Hybrid Surface/Voume Segmentation in MRtrix3 5tt format. Aligned to the FreeSurfer ``orig.mgz`` image.
+\*space-T1w\*seg-hsvs\*_dseg.nii.gz,Hybrid Surface/Voume Segmentation in MRtrix3 5tt format. Aligned in coordinate space to ``space-T1w``. 
+\*space-fsnative\*seg-hsvs\*_dseg.nii.gz,Hybrid Surface/Voume Segmentation in MRtrix3 5tt format. Aligned to the FreeSurfer ``orig.mgz`` image.
```

Comment:
```suggestion
\*space-T1w\*seg-hsvs\*_dseg.nii.gz,Hybrid Surface/Volume Segmentation in MRtrix3 5tt format. Aligned in coordinate space to ``space-T1w``. 
\*space-fsnative\*seg-hsvs\*_dseg.nii.gz,Hybrid Surface/Volume Segmentation in MRtrix3 5tt format. Aligned to the FreeSurfer ``orig.mgz`` image.
```

If this is a surface/volume segmentation, I don't think `nii.gz` is the right extension. Hybrid surface/volume says CIFTI to me.

Reply:
it's a nifti - they use the surface meshes to fill in voxels